src/styles/blocks/cards.scss

```
@@ -0,0 +1,88 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  width: 200px;
+  &__review {
+    display: flex;
+    justify-content: space-between;
```

Better add page break.
```suggestion

  width: 200px;
  
  &__review {
    display: flex;
    justify-content: space-between;
```